ggshield/sca/output/text_handler.py

```diff
@@ -73,9 +98,16 @@ def process_sca_file_result(
 
 def no_sca_vulnerabilities() -> str:
     """
-    Build a message if no IaC vulnerabilities were found.
+    Build a message if no SCA vulnerabilities were found.
+    """
+    return format_text("\nNo SCA vulnerability has been found.\n", STYLE["no_secret"])
+
+
+def no_sca_vulnerability_added() -> str:
+    """
+    Build a message if no SCA vulnerabilities were added.
     """
-    return format_text("\nNo SCA vulnerability has been found\n", STYLE["no_secret"])
+    return format_text("\nNo SCA vulnerability added.\n", STYLE["no_secret"])
```

Comment:
```suggestion
    return format_text("\nNo added SCA vulnerabilit.\n", STYLE["no_secret"])
```

or No SCA vulnerability has been added.